gvm/protocols/gmpv7.py

```diff
@@ -1935,40 +1935,284 @@ def modify_filter(self, filter_id, comment=None, name=None, term=None,
+            hosts (list, optional): A list of hosts addresses
+            port (str, optional): Port to which note applies.
+            result_id (str, optional): Result to which note applies.
+            severity (str, optional): Severity to which note applies.
+            task_id (str, optional): Task to which note applies.
+            threat (str, optional): Threat level to which note applies.
+        """
+        if not note_id:
+            raise RequiredArgument('modify_note requires a note_id attribute')
+        if not text:
+            raise RequiredArgument('modify_note requires a text element')
+
+        cmd = XmlCommand('modify_note')
+        cmd.set_attribute('note_id', note_id)
+        cmd.add_element('text', text)
+
+        if not active is None:
+            cmd.add_element('active', '1' if active else '0')
+
+        if hosts:
+            cmd.add_element('hosts', ', '.join(hosts))
+
+        if port:
+            cmd.add_element('port', port)
+
+        if result_id:
+            cmd.add_element('result', attrs={'id': result_id})
+
+        if severity:
+            cmd.add_element('severity', severity)
+
+        if task_id:
+            cmd.add_element('task', attrs={'id': task_id})
+
+        if threat:
+            cmd.add_element('threat', threat)
+
+        return self._send_xml_command(cmd)
+
+    def modify_override(self, override_id, text, active=None, hosts=None,
+                        port=None, result_id=None, severity=None,
+                        new_severity=None, task_id=None, threat=None,
+                        new_threat=None):
+        """Generates xml string for modify override on gvmd.
+
+        Arguments:
+            override_id (str): UUID of override to modify.
+            text (str): The text of the override.
+            active (int, optional): Seconds override will be active.
+                -1 on always, 0 off.
+            hosts (list, optional): A list of host addresses
+            port (str, optional): Port to which override applies.
+            result_id (str, optional): Result to which override applies.
+            severity (str, optional): Severity to which override applies.
+            new_severity (str, optional): New severity score for result.
+            task_id (str, optional): Task to which override applies.
+            threat (str, optional): Threat level to which override applies.
+            new_threat (str, optional): New threat level for results.
+        """
+        if not override_id:
+            raise RequiredArgument('modify_override requires a override_id '
+                                   'argument')
+        if not text:
+            raise RequiredArgument('modify_override requires a text argument')
+
+        cmd = XmlCommand('modify_override')
+        cmd.set_attribute('override_id', override_id)
+        cmd.add_element('text', text)
+
+        if not active is None:
+            cmd.add_element('active', '1' if active else '0')
```

Comment:
Same here. I am sorry!!!
```suggestion
            cmd.add_element('active', seconds_active)
```